README.md

```diff
@@ -21,6 +21,23 @@ application is sufficient to make the module available to your application.
 
 TODO: -->
 
+## Description
+
+ -Module provies wrapper to a credentials store (bedrock-credentials-mongodb for example).
+
+ -Angular module provides directive for credentials, and provides routes that map to templates that have a controller that will
+ either store or transmit the credentials using navigator.credentials.transmit to a credential consumer.
+
+ -Can compose a view of an identity.
+
+## Dependencies:
+ - bedrock-angular-identity-composer
+ - bedrock-angular-credential
```

Comment:
also, make sure to specify the "Bedrock npm dependencies" and the "Bedrock bower dependencies"... and specify why the dependency exists. For example: "bedrock-angular-credential" - "Used to display and manipulate credentials..." (I'm not sure that's what it does, but hopefully you get the idea).